casper/src/main/scala/coop/rchain/casper/dag/BlockDagKeyValueStorage.scala

```diff
@@ -297,4 +313,34 @@ object BlockDagKeyValueStorage {
       stores.deployPool,
       stores.invalidBlocks
     )
+
+  def deployStatus[F[_]: Sync](
+      d: DeployId,
+      deployStore: KeyValueTypedStore[F, DeployId, Signed[DeployData]],
+      getRepresentation: F[DagRepresentation],
+      deployIndex: KeyValueTypedStore[F, DeployId, BlockHash],
+      vabnOpt: DeployId => Option[Long],
```

Comment:
What does it mean optional result for vabn?

Reply:
I used inmem map to track VABN, so to not read deploy store each time.
I adjusted logic to not use it bit read after each block is added, so this is removed <https://github.com/rchain/rchain/pull/3682/commits/51f96ca8ba170ca4f4c2c8d6ac9aae41949b8cf3>.